docs/whats-new/staging-release-notes.md

```diff
@@ -1,5 +1,22 @@
 # Staging release notes
 
+## November 2023
+
+**09 November 2023**
+
+Frontend version: 1.0.0-20231102.1605 | Backend version: 1.71.2-20231108.1413
+
+| Type      | Change      | Description |
+| --- | --- | --- |
+| **Feature** |  Automated account termination | We have automated the account termination process for employees who have left the organization in our [service request form](https://go.gov.sg/seed-techpass-support). <br><br><b>Note</b>: Automation applies only to cases initiated by a Public officer (for another user) or when the user themselves terminates their own account, provided the user is not onboarded to SEED.|
```

Comment:
- how come the Public officer has a capital P ?
- how come the **for another user** is in the parenthesis?

Reply:
@JLeeChan ah sorry. because in TechBiz, the UI for PO is Public officers so i probably auto wrote is Public officers. will change.

will remove the paranthesis